pkg/render/logstorage_test.go

```diff
@@ -1609,7 +1609,7 @@ var deleteLogStorageTests = func(managementCluster *operatorv1.ManagementCluster
 }
 
 func compareResources(resources []client.Object, expectedResources []resourceTestObj) {
-	Expect(len(resources)).To(Equal(len(expectedResources)))
+	// First compare item-by-item, as it provides clear error messages.
```

Comment:
The test will probably panic if expectedResources is longer than resources. How is a panic better than 1st a message that the len of the 2 is different. If we want to see the items in the lists when the len fails then please add to output resources and expectedResources on failure of the len equality check.